playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -377,6 +377,7 @@ EDXAPP_BUGS_EMAIL: 'bugs@example.com'
 EDXAPP_DEFAULT_FROM_EMAIL: 'registration@example.com'
 EDXAPP_DEFAULT_FEEDBACK_EMAIL: 'feedback@example.com'
 EDXAPP_DEFAULT_SERVER_EMAIL: 'devops@example.com'
+EDXAPP_STANFORD_COPYRIGHT_EMAIL = 'copyright@example.com'
```

Comment:
This name is too specific; it shouldn't say `Stanford`.

Note: This comment applies everywhere this variable or some form thereof is used.